Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -93,7 +93,7 @@ public Analytics(Config config, ImportedReferences references, ILogger logger, M
 
             _references = references;
             _logger = logger;
-
+            _realmEnvMetrics = new();
```

Comment:
Why was that needed?

Reply:
All other fields were initialized in the constructor (because they have to), so just for consistency I brought this there to have all initialization in 1 place.
If we have rules on style about this, I'll revert.

Reply:
No, we don't have strict rules on that - I generally prefer to initialize fields wherever they are declared if possible - it saves a line of code and makes it more obvious, especially with the target-typed ctor (since you have the type on the same line). I'm fine either way, was just surprised to see this and thought I'm missing something.